README.md

```diff
@@ -47,6 +47,61 @@ Instructions
 9. Load the firmware onto the motor board:
 
 	$ stm32flash -w mcv4.bin -v /dev/ttyUSB<X>
+	
+## Controls
+
+The motor board is largely controlled over the serial interface. There is one physical push button on the board that is a firmware reset button.
```

Comment:
It should be noted that pressing it actually reboots the board into a different mode, in which a new firmware image can be loaded. It's probably referred to as the reset button a lot because the way this behaviour is achieved is to reset the microprocessor while certain pins are held in certain states.

```suggestion
The motor board is largely controlled over the serial interface. There is one physical push button on the board that puts the microprocessor into a mode in which new firmware can be installed.
```

(I know this is probably copied verbatim from the old source, but its probably worth making the fix now).